docs/developer-guide/emt-contribution.md

```diff
@@ -19,10 +19,33 @@ review, better understand, and prioritize the request.
 
 ## Contribution Flow
 
-The following diagram outlines the general workflow for pull requests made
-to the Edge Microvisor Toolkit repository:
+All contributing Pull Requests should target the **3.0-dev branch**, which is used for
+development. Periodically, the dev branch is merged to 3.0, the publishing branch for the
+project.
+
+Following the [Azure Linux](https://github.com/microsoft/azurelinux/blob/3.0/CONTRIBUTING.md#packages)
+scheme for packages, Edge Microvisor Toolkit maintains two directories:
+* [SPECS](../../SPECS) - fully supported and published,
+* [SPECS-EXTENDED](../../SPECS-EXTENDED) - experimental, if you want to use these packages,
+  you will need to [run your own build](./get-started/emt-building-howto.md).
+
+When raising a PR, make sure that:
+
+* The [cgmanifest file](./cgmanifest.json) is up-to-date and alphabetically sorted.
+  It is used to record all package sources. To validate it, run:
+
+  ```
+  .github/workflows/validate-cg-manifest.sh SPECS/<package name>/<package-name>.spec
+  ```
+
+* All source files have up-to-date hashes in the *.signatures.json files.
+  This hash can be generated by sha256sum or by using a make target:
+
+  ```
+  cd toolkit
+  sudo make input-srpms REBUILD_TOOLS=y SRPM_FILE_SIGNATURE_HANDLING=update
+  ```
 
-![Contribution Flow](assets/emt-contribution-flow.drawio.svg)
```

Comment:
Should we remove this file as well from tree? I will merge it for now. Please remove in a follow up PR if you agree.